conans/client/store/localdb.py

```diff
@@ -2,6 +2,7 @@
 from conans.errors import ConanException
 
 USER_TABLE = "users"
```

Comment:
Maybe remove this global variable USER_TABLE and put it in the DROP statement. (with a comment: Old user table, now user and tokens are associated to a remote)